.travis/travis-install-dependencies

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+# Modified version of
+# https://github.com/ckan/ckan/blob/master/bin/travis-install-dependencies
+
+set -e
+set -x
+
+# Drop Travis' postgres cluster if we're building using a different pg version
+TRAVIS_PGVERSION='9.1'
+
+if [ ${PGVERSION} != ${TRAVIS_PGVERSION} ]
+then
+  sudo -u postgres pg_dropcluster --stop ${TRAVIS_PGVERSION} main
+fi
+
+# Install PostgreSQL from apt.postgresql.org
+sudo apt-get update -qq
+sudo apt-get install postgresql-${PGVERSION} postgresql-contrib-${PGVERSION}
+
+if [ ${PGVERSION} = '8.4' ]
```

Comment:
We're exploring adding support for that version and contributing it back.  (Hurray for legacy applications...)